### PR TITLE
Created API and files to deploy to Heroku to test API

### PR DESCRIPTION
**Pull request by CodingPenguin**
PLS APPROVE LIKE A BOSS

### CODE REVIEW

File: sentysis/sentysis.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    #   "youtube_url": "https://www.youtube.com/watch\?v\=1ecG3MT0lAw"
+    #   }
+    }
+]
+
+
+
+@app.route('/')
+def create_UI():
+    return "<h1>WHAT'S UP LOSER</h1>"
```
Comment (leevanielle) [inline sentysis/sentysis.py:22]: LOL

File: sentysis/sentysis.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+@app.route('/api/ytUrls', methods=['GET', 'POST'])
```
Comment (leevanielle) [inline sentysis/sentysis.py:24]: You don't need `'GET'`

**Review by leevanielle — CHANGES_REQUESTED** [review]

File: sentysis/sentysis.py (hunk 2 above)
Comment (CodingPenguin) [reply sentysis/sentysis.py:24]: Ok yah got it

**Comment by leevanielle** [comment]
Next time don't merge until I approve. Thanks!

**Pull request merged**